### DISCUSSION TITLE
Rename id watermeter for the homeassistant integration

### DISCUSSION THREAD

**Discussion by guckmaster** [jomjol/AI-on-the-edge-device#3133] [General]
I have an old esp with watermeter Entities in homeassistant. They are nit avialable.
I want to connect a new esp with the sane same Id watermeter. 
but i cant delete it in the inegation.

**Comment by caco3** [comment]
the MQTT ID can be changed in the expert settings!
See https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-clientid
If you refer to the main topic, that can also be changed, see https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-maintopic
